src/components/tabs/CdrTabs.vue

```diff
@@ -26,11 +25,17 @@ const props = defineProps({
 })
 
 const slots = useSlots();
-const slottedTabs = slots.default()[0].children?.length ? slots.default()[0].children : slots.default();
+const slottedTabs = slots.default()[0].children?.length
+  ? slots.default()[0].children
+  : slots.default();
 const baseClass = 'cdr-tabs';
 
 //Refs
-const tabs = ref(slottedTabs.map((tab) => ({ name: tab.props.name, disabled: tab.props.disabled })));
+const tabs = ref(slottedTabs.map((tab) => ({ 
+  name: tab.props.name,
+  disabled: tab.props.disabled,
+  id: tab.props['aria-labelledby'] 
```

Comment:
Without using the `aria-labelledby` (as recommended here https://rei.github.io/rei-cedar-docs/components/tabs/#cdrtabpanel), CdrTabsPanel was not generating unique IDs

Reply:
This also preserves how the Vue 2 version worked